todo.txt

```diff
@@ -7,6 +9,30 @@ x managing health should be done by the health class
 - add enemy types
 x enemies are able to slip through walls somehow
 
+editor stuff
+- resizable main window
+x switch edit mode from keybinds (decouple edit mode from gui, so we can change the gui to new layout)
+~ Visual indicator for edit mode
+    - Text
+    ~ Cursor
+- Erase tool should not have to collide with with the texture game object to delete the texture. 
```

Comment:
I can't remember if all objects within the same tile and layer are deleted at once with the erase tool, but they should if they don't

Reply:
Current behavior is that it the erase tool doesn't delete all object's on the same tile and layer. I'll fix that with the next PR.